cvat/apps/lambda_manager/apps.py

```diff
@@ -3,3 +3,7 @@
 
 class LambdaManagerConfig(AppConfig):
     name = 'cvat.apps.lambda_manager'
+
+    def ready(self) -> None:
```

Comment:
What do you think about cvat.apps.iam.permissions.AppConfigEx class with implemented ready() method by default? We can try to handle several additional modules such as signals.py and permissions.py. All apps have to use AppConfigEx.

Reply:
IMO, a class that loads both signals and permissions isn't a very compelling abstraction. Signals have nothing to do with permissions, or with IAM as a whole. An app may only use one of them (in fact, the app on which you posted this comment does not use signals).

I suppose I could add two separate mixin classes, so you could have something like:

```python
class EngineConfig(AppConfig, IamAppMixin, SignalAppMixin):
    name = 'cvat.apps.engine'
```

but the amount of code saved in this approach is pretty marginal.